src/flows/context/flow.list.tsx

```diff
@@ -173,6 +177,8 @@ export const FlowListProvider: FC = ({children}) => {
         readOnly: false,
         range: DEFAULT_TIME_RANGE,
         refresh: AUTOREFRESH_DEFAULT,
+        createdAt: new Date().toISOString(),
+        updatedAt: new Date().toISOString(),
```

Comment:
not sure about applying defaults here. This should be from the API but we create and set the flow on the UI so maybe?

Reply:
yeah i think we should skip the defaults and conditionally render? like we're telling the user "this was last saved at XYZ" and if it was never saved......

Reply:
Right totally. I guess by the time they actually access the data (after creation) that data will already be saved so it doesn't really make sense